common/network-common/src/main/java/org/apache/spark/network/buffer/FileSegmentManagedBuffer.java

```diff
@@ -132,7 +134,8 @@ public Object convertToNetty() throws IOException {
     if (conf.lazyFileDescriptor()) {
       return new DefaultFileRegion(file, offset, length);
     } else {
-      FileChannel fileChannel = new FileInputStream(file).getChannel();
+      FileChannel fileChannel = FileChannel.open(file.toPath(),
+        ImmutableSet.of(StandardOpenOption.READ));
```

Comment:
They're actually the same, the one you mentioned is just a simple wrapper of the former one. I will change to yours for the simplicity.